istio/src/main/scala/io/buoyant/k8s/istio/IstioServices.scala

```diff
@@ -0,0 +1,43 @@
+package io.buoyant.k8s.istio
+
+import com.twitter.finagle.buoyant.H2
+import com.twitter.finagle.{Address, Name}
+import com.twitter.logging.Logger
+import io.buoyant.config.types.Port
+import io.buoyant.k8s.istio.mixer.MixerClient
+
+object IstioServices {
+  protected val log = Logger()
+
+  def mkMixerClient(mixerHost: Option[String], mixerPort: Option[Port]) = {
```

Comment:
I like explicit return types on public methods

Reply:
Me too, will fix it. I think Intellij has a setting for this somewhere?